leaflet-treering.js

```diff
@@ -835,6 +839,12 @@ function VisualAsset (Lt) {
       } else {
         var color = '#00838f';
       }
+
+      //mark decades with red line
+      if ((pts[i].earlywood && pts[i].year % 10 == 0) || (!pts[i].earlywood && pts[i].year % 10 == 0)) {
```

Comment:
I think this conditional just reduces down to if(pts[I].year %10 ==) right? Do you need to check earlywood and !earlywood?

Reply:
I don't. I originally wrote it for just latewood and reused code without thinking too much about it.